apps/web/src/views/Swap/SmartSwap/index.tsx

```diff
@@ -183,12 +189,16 @@ export const SmartSwapForm: React.FC<{ handleOutputSelect: (newCurrencyOutput: C
         [Field.INPUT]:
           independentField === Field.INPUT
             ? parsedAmount
+            : isMMLoading
```

Comment:
Not sure if this will cause issue when MM loading, and AMM have result, but we just show undefined for a while?

Reply:
We should wait for MM complete

Reply:
Got it👌